packages/nouns-webapp/src/wrappers/nounsAuction.ts

```diff
@@ -1,6 +1,6 @@
 import { useContractCall } from '@usedapp/core';
 import { BigNumber as EthersBN, utils } from 'ethers';
-import { Contract } from '@ethersproject/contracts';
+import { Contract } from '@usedapp/core/node_modules/ethers';
```

Comment:
Hmm shouldn't need to do this. The types are compatible now because both are on version 5.4.x. May need to nuke your local node modules and run yarn if you're getting type errors.

Reply:
You're right - nuked node modules and it worked 👍